### PR TITLE
Hacktoberfest blog post

### PR DESCRIPTION
**Pull request by jenweber**
This is a blog post to announce Hacktoberfest!

Times editors, I tagged you for feedback! This will not be an Ember Times thing but I'd love to know if you think I'm missing anything.

### CODE REVIEW

File: source/2020-10-01-hacktoberfest.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Good news! It's Hacktoberfest 2020, and you are invited to participate in Ember-focused special events and mentorship opportunities through the month of October.
+You can make some commits, meet some new people, and maybe get some free stuff!
+
+## What is Hacktoberfest?
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is an annual, worldwide celebration organized by DigitalOcean and sponsored by many companies that use open source software.
+
+The premise is simple: 
+- Sign up anytime between October 1 and October 31
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]: Due to a Markdown linter that we have, I think you need to add a line before starting a list.

```suggestion
The premise is simple: 

- Sign up anytime between October 1 and October 31
```

File: source/2020-10-01-hacktoberfest.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Make four valid Pull Requests to _any_ public repositories on GitHub
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]: For small numbers, I think it's easier to read the number.

```suggestion
- Make 4 valid Pull Requests to _any_ public repositories on GitHub
```

File: source/2020-10-01-hacktoberfest.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- If you are among the first 70,000 developers who complete the challenge, you get a prize. This year's prizes include T-shirts or a tree planted in your name.
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]: Maybe:
```suggestion
- If you are among the first 70,000 developers who complete the challenge, you get a prize. This year, you can select between a T-shirt and a tree planted in your name.
```
?

File: source/2020-10-01-hacktoberfest.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Many open source project maintainers put in some extra effort during Hacktoberfest to make it easy and fun to participate in open source.
+
+## What is happening in the Ember community?
+
+Here's what you can expect this year!
+
+- **On Saturday, October 10th,** past contributors will be available to pair with anyone who wants to help out! If you have questions or need some help finding an issue to work on, this is a great place to start. Specific times will be announced soon.
+- **On Friday, October 16th,** there will be an Ember Contributors Workshop! DESCRIPTION AND LINK
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]: Reminder to add description and link before merging PR.

File: source/2020-10-01-hacktoberfest.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- **Any time,** join the `#hacktoberfest` channel in the [Ember Community Discord]() to connect with other developers. You can find a buddy to pair with, a mentor to guide you, or toss around ideas.
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]:
```suggestion
- **At any time,** join the `#hacktoberfest` channel in the [Ember Community Discord](https://discord.gg/emberjs) to connect with other developers. You can find a buddy to pair with, a mentor to guide you, or toss around ideas.
```

File: source/2020-10-01-hacktoberfest.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+If you are someone who is willing to help mentor new contributors, please let us know in `#hacktoberfest` on Discord. Addon maintainers are welcome to post issues they need help with too!
+
+Additionally, there are several more groups within the Ember Community Discord where you can find others to collaborate with. There are Discord channels for women, lgbtq+, and people of color. If you belong to one of those groups, you can ask to join in the `#discord-server-admin` channel.
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]:
```suggestion
Additionally, there are several more groups within the Ember Community Discord where you can find others to collaborate with. There are Discord channels for women, LGBTQ+, and people of color. If you belong to one of those groups, you can ask to join in the `#discord-server-admin` channel.
```
In addition to asking publicly in the admin channel, can we provide people the option to privately message a mod (maybe @MelSumner if, Melanie, you are okay with being pinged)?

File: source/2020-10-01-hacktoberfest.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+## How do I choose what to work on?
+
+It all depends on your interests! One of the best ways to get involved in open source is to pick up a teeny tiny issue so you can learn the flow and what to expect. Then, consider making an improvement to something you regularly use, for example, an addon feature, test coverage for a past bug, or fixes on some page of API documentation.
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]: For an optional suggestion, I think we can shorten the last sentence.
```suggestion
It all depends on your interests! One of the best ways to get involved in open source is to pick up a teeny tiny issue so you can learn the flow and what to expect. Then, consider making an improvement to something you regularly use; for example, a feature, test, or fix to API documentation.
```

File: source/2020-10-01-hacktoberfest.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+For ideas, you can visit [Ember's Help Wanted](https://help-wanted.emberjs.com/). The Ember Learning Team and others are curating some super beginner-friendly issues, and you can find them there, along with more intermediate and advanced issues. If you want to help, comment on an issue to let others know that you are working on it. If the issue you want to work on is taken, try asking if the other volunteers would like to collaborate!
```
Comment (ijlee2) [inline source/2020-10-01-hacktoberfest.md]:
```suggestion
For ideas, you can visit [Ember's Help Wanted](https://help-wanted.emberjs.com/). The Ember Learning Team and others are curating some super beginner-friendly issues. In Help Wanted, you can find these issues, along with more intermediate and advanced issues. If you want to help, comment on an issue to let others know that you are working on it. If the issue you want to work on is taken, try asking if the other volunteers would like to collaborate!
```

**Review by ijlee2 — CHANGES_REQUESTED** [review]
@jenweber The announcement reads great overall. I appreciate your working on the writeup and community events! I left a few suggestions that I think can help with the writing.

I noticed that `alex` linter marked a few words as insensitive (why I asked for a change). Can you check these words to see if they are okay in the context of your writing? You can run `alex source/2020-10-01-hacktoberfest.md` locally (after you install `alex`), then add lint exceptions if you need to.

File: source/2020-10-01-hacktoberfest.md (hunk 2 above)
Comment (amyrlam) [inline source/2020-10-01-hacktoberfest.md]:
```suggestion
- Make four valid Pull Requests to <span style="font-style: italic;">any</span> public repositories on GitHub
```

this repo doesn't honor markdown italic. we list it as a "gotcha" on the latest ember times post description (https://github.com/ember-learn/ember-blog/pull/778), is my soln for now

Comment (mansona) [reply source/2020-10-01-hacktoberfest.md]: 🙈  at least the new redesign will support it in a few weeks 😫  (sorry for the delay)

Comment (cah-danmonroe) [reply source/2020-10-01-hacktoberfest.md]: For small numbers, it is grammatically correct for them to be spelled out.

**Review by mansona — APPROVED** [review]
A great post 🎉  It will be great to get this out there 👍

**Review by ijlee2 — APPROVED** [review]
Thank you!

**Pull request merged**